### PR TITLE
Update AppInfo to include content descriptors

### PR DESCRIPTION
**Pull request by AbdulDridi**
## Summary

One thing I wanted to raise was that there's a slight divergence between what the ContentDescriptor ids represent between this and SteamKit - I didn't refactor because I didn't want to introduce a breaking change - could be worth a refactor though?

Screenshot 2024-04-04 at 13 15 33

## Checklist

- [ ] If code changes were made then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by Gobot1234** [comment]
It might be worth making `FrequentNudityOrSexualContent` just `NudityOrSexualContent` and adding an alias for the old one.

File: steam/manifest.py [hunk 1]
```diff
@@ -924,6 +934,10 @@ def __init__(
         self._stats_visible = common.get("community_visible_stats", "0") == "1"
         self._free = extended.get("isfreeapp", "0") == "1"
 
+        self.content_descriptors = [
+            ContentDescriptor.try_value(int(id)) for id in common.get("content_descriptors", MultiDict()).values()
+        ]
```
Comment (Gobot1234) [inline steam/manifest.py:939]: Please update the types.manifest file for this and add a small docstring for it

Comment (AbdulDridi) [reply steam/manifest.py:939]: done

**Comment by Gobot1234** [comment]
Awesome thanks for this

**Pull request merged**